go/src/mdtf-public/rally2-matching-system/README.md

```diff
@@ -0,0 +1,7 @@
+# Sample Biometric Technology Rally II Matching System
+
+This folder contains a simple application, written in [The Go Programming Language](https://golang.org/).  This application implements the [Rally Matching System API](https://github.com/TheMdTF/mdtf-public/blob/master/api/rally2-matching-system/README.md).  Creating an application that implements this API is required for matching system providers to participate in the [2019 Biometric Technology Rally](https://mdtf.org/Rally2019).  Finished applications should be packaged and saved using [Docker](https://www.docker.com/).  Scripts that package and save this application are in the `rally2-matching-system/docker` folder.
+
+It is not required that matching system providers implement the matching system API in Go, as the application in this folder does.  Matching system providers are free to use whatever technology stack they choose, as long as the final docker container conforms to the API.  Scripts that send API conformant requests to a user provided endpoint are provided in the `rally2-matching-system/test` folder.
+
+To run the sample application using Go, set your `GOPATH` to `mdtf-public/go`.  The only external dependencies for this application are the [Go Disintegration Imaging](https://godoc.org/github.com/disintegration/imaging) package, which can be installed via `go get https://github.com/disintegration/imaging`.  The application can then be run via `go run main.go`.  It can be compiled via `go build -o ./rally2-mathcing-system`.  Once the application has been built, it can be packaged using the scripts in `rally2-matching-system/docker` and tested using the scripts in `rally2-matching-system/test`.
```

Comment:
Need to add a note about building the C wrapper prior to trying to run/build the service.

Reply:
You should probably use a build image to build binary inside of the image rather than making people install dependencies on their machine

Reply:
Excellent idea, please see the Issue at https://github.com/TheMdTF/mdtf-public/issues/6.  We eagerly await your PR.